capella2polarion/converters/link_converter.py

```diff
@@ -71,6 +71,13 @@ def create_links_for_work_item(
                     new: cabc.Iterable[str]
                     if isinstance(refs, common.ElementList):
                         new = refs.by_uuid  # type: ignore[assignment]
+                    elif refs is None:
+                        warning = make_link_logging_message(
+                            "No model element behind that attribute",
+                            link_config.capella_attr,
+                        )
+                        link_errors.extend(warning)
+                        continue
```

Comment:
```suggestion
                    elif refs is None:
                        logger.info("For model element %r attribute %s is not set", obj._short_repr_(), link_config.capella_attr)
                        continue
```
It's perfectly fine, if an attribute is None, there is no reason to log a warning in IMO.